examples/data/priorApproval/largerExtension.ts

```diff
@@ -0,0 +1,1546 @@
+          },
+          {
+            value: 'registeredPark',
+            description: 'Historic Park or Garden',
+            intersects: false,
+          },
+          {
+            value: 'designated.AONB',
+            description: 'Area of Outstanding Natural Beauty (AONB)',
+            intersects: false,
+          },
+          {
+            value: 'nature.ramsarSite',
+            description: 'Ramsar site',
+            intersects: false,
+          },
+          {
+            value: 'designated.nationalPark',
+            description: 'National Park',
+            intersects: false,
+          },
+          {
+            value: 'designated.conservationArea',
+            description: 'Conservation Area',
+            intersects: false,
+          },
+          {
+            value: 'designated.nationalPark.broads',
+            description: 'National Park - Broads',
+            intersects: false,
+          },
+          {
+            value: 'road.classified',
+            description: 'Classified Road',
+            intersects: false,
+          },
+        ],
+      },
+      boundary: {
+        site: {
+          type: 'Feature',
+          geometry: {
+            type: 'MultiPolygon',
+            coordinates: [
+              [
+                [
+                  [-0.072763, 51.456622],
+                  [-0.072749, 51.456669],
+                  [-0.073167, 51.456732],
+                  [-0.073195, 51.456736],
+                  [-0.073213, 51.456688],
+                  [-0.072763, 51.456622],
+                ],
+              ],
+            ],
+          },
+          properties: {
+            name: '',
+            entity: 12000593377,
+            prefix: 'title-boundary',
+            dataset: 'title-boundary',
+            'end-date': '',
+            typology: 'geography',
+            reference: '37786766',
+            'entry-date': '2024-05-06',
+            'start-date': '2002-06-26',
+            'organisation-entity': '13',
+          },
+        },
+        area: {
+          hectares: 0.017289,
+          squareMetres: 172.89,
+        },
+      },
+    },
+    application: {
+      type: {
+        value: 'pa.part1.classA',
+        description: 'Prior Approval - Larger extension to a house',
+      },
+      fee: {
+        calculated: 120,
+        payable: 0,
+        exemption: {
+          disability: true,
+          resubmission: false,
+        },
+        reduction: {
+          sports: false,
+          parishCouncil: false,
+          alternative: false,
+        },
+      },
+      declaration: {
+        accurate: true,
+        connection: {
+          value: 'none',
+        },
+      },
+    },
+    proposal: {
+      projectType: [
+        {
+          value: 'extend.rear',
+          description: 'Add a rear or side extension (or conservatory)',
+        },
+      ],
+      description:
+        'A 2 storey rear extension with a roof garden and built in pizza oven',
```

Comment:
Good spot here @jessicamcinchak (re: https://github.com/theopensystemslab/digital-planning-data-schemas/pull/181#discussion_r1656649100)

I've raised the question on Slack for the content team to address - it does appear that this is not correctly set in PA flows. https://opensystemslab.slack.com/archives/C5Q59R3HB/p1720004192950049

Reply:
Manually setting it here is great then, I did the same across other PA examples.